ops-bedrock/fork.docker-compose.yml

```diff
@@ -0,0 +1,184 @@
+version: '3.8'
+
+# This Compose file is expected to be used with the devnet-up.sh script.
+# The volumes below mount the configs generated by the script into each
+# service.
+
+volumes:
+  l1_data:
+  l2_data:
+  op_log:
+
+
+services:
+  op_stack_go_builder:  # Not an actual service, but builds the prerequisite go images
+    build:
+      context: ../
+      dockerfile: ops/docker/op-stack-go/Dockerfile
+      args:
+        GIT_COMMIT: "dev"
+        GIT_DATE: "0"
+    image: onthertech/titan-op-stack-go:${IMAGE_TAG:-latest}
+    entrypoint: ["echo", "build complete"]
+
+  l1:
+    build:
+      context: .
+      dockerfile: Dockerfile.l1-fork
+    ports:
+      - "8545:8545"
+    volumes:
+      - "${PWD}/../.devnet/genesis-l1.json:/genesis.json"
+    environment:
+      GETH_MINER_RECOMMIT: 100ms
+      L1_RPC: ${L1_RPC}
+      FROM_BLOCK_NUMBER: ${FROM_BLOCK_NUMBER}
+
+  fork-l1:
```

Comment:
The copy of docker-compose.yml but adding a new one for generating the data in the genesis file